### PR TITLE
★★★ DB Updates4u ★★★

### PR DESCRIPTION
**Pull request by hiwaylon**
# ★

★★★

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -26,10 +26,6 @@
       android:theme="@style/AppTheme"
       tools:ignore="AllowBackup,GoogleAppIndexingWarning">
     <activity android:name=".ChecklistActivity">
-      <intent-filter>
```
Comment (aaronj1335) [inline app/src/main/AndroidManifest.xml:29]: this shouldn't have changed

**Comment by aaronj1335** [comment]
other than the manifest change, :shipit: 

feel free to update and push, no second look needed.

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (hiwaylon) [inline app/src/main/AndroidManifest.xml:29]: GONK! My bad.

Comment (hiwaylon) [inline app/src/main/AndroidManifest.xml]: GAK. My bad.

**Pull request merged**